File: src/com/sparkSQL/demo03/Person.java
package com.sparkSQL.demo03;

import java.io.Serializable;

public class Person implements Serializable {
    public static String isPerson = "";
    public String id;
    public String name;
    public String age;

    public static String getIsPerson() {
        return isPerson;
    }

    public static void setIsPerson(String isPerson) {
        Person.isPerson = isPerson;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }
}

File: src/com/scala/Homework/Homework04/Person.scala
package com.scala.Homework.Homework04

class Person {
  private var age: Int = 0
  private var name: String = ""

  var stu = new Student

  def setGrade(grade:Int) = stu.setgrade_=(grade)
  def getGrade = stu.getgrade

  def setProfession(pro:String) = stu.setProfession_=(pro)
  def getProfession = stu.getProfession

  def setSchoolName(sn:String) = stu.setSchoolName_=(sn)
  def getSchoolName = stu.getSchoolName

  def getage = age

  def setage_=(newAge: Int): Unit = {
    age = newAge
  }

  def getname = name

  def setname_=(newName: String): Unit = {
    name = newName
  }

  def printPerson(): Unit ={
    val g = getGrade
    val p = getProfession
    val s = getSchoolName

    printf("age=%d, name=%s, grade=%d, profession=%s, schoolName=%s",age,name,g,p,s)
  }

}
